### PR TITLE
Remove console.log when looking up non-default VPCs

### PR DESCRIPTION
**Pull request by samjarrett**
*Description of changes:*

A debug `console.log()` seems to have been left in place that causes LookedUpVpc objects to be logged out when using imported (non-default) VPC/Subnets. This change removes that.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by elamaran11 — COMMENTED** [review]
LGTM

**Comment by samjarrett** [comment]
Hi team, how do we get this moving?

**Comment by shapirov103** [comment]
I am happy to merge. The change is fairly minor, so once merged it will wait for the next release.

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**
